src/ReactSelectMe.js

```diff
@@ -238,6 +238,12 @@ export default class ReactSelectMe extends PureComponent {
         onFocus={this.onSearch}
         onClick={this.onSearch}
         onPaste={this.onSearch}
+        onKeyUp={(e) => {
+          /* Internet Explorer detection. See more
+           * http://stackoverflow.com/questions/9847580/how-to-detect-safari-chrome-ie-firefox-and-opera-browser
+           */
+          if (false || !!document.documentMode) this.onSearch(e);
```

Comment:
`false` is always false, isn't it? :) Can we get rid of it?

I would also suggest you move this logic to the `onSearch` method.

So here you will have 
```
onKeyUp={this.onSearch}
```

And inside the `onSearch` method:
```javascript
onSearch = (evt) => {
  if (evt.type === 'keyup' && !!document.documentMode) {
    return;
  }
  // ...
```